bumble/device.py

```diff
@@ -1706,6 +1988,18 @@ def find_connection_by_bd_addr(
 
         return None
 
+    def lookup_periodic_advertising_sync(
```

Comment:
This could be unnecessary if PA table is a dict mapped from handle?

Reply:
That wouldn't work, because the `periodic_advertising_syncs` list contains entries that don't have a handle yet (the handle is only assigned later when the controller sends a sync establishment event)